src/main/java/com/google/maps/DistanceMatrixApiRequest.java

```diff
@@ -19,12 +19,7 @@
 
 import com.google.maps.DirectionsApi.RouteRestriction;
 import com.google.maps.DistanceMatrixApi.Response;
-import com.google.maps.model.DistanceMatrix;
-import com.google.maps.model.LatLng;
-import com.google.maps.model.TransitMode;
-import com.google.maps.model.TransitRoutingPreference;
-import com.google.maps.model.TravelMode;
-import com.google.maps.model.Unit;
+import com.google.maps.model.*;
```

Comment:
Can you please revert this change? We don't used star imports, per the Google Java style guide lines.